### ISSUE TITLE
REPORT requests with absolute URL fail with Radicale running on Windows

### ISSUE THREAD

**Issue by Ede123** [Kozea/Radicale#192]
There's a bug with REPORT requests with Radicale running on Windows:
- If the `href` attribute of the request contains a (valid) absolute URL to an item Radicale sends back an empty response.
- If the `href` attribute of the request contains a relative URL  (even invalid) to an item Radicale sends back the expected response.

To reproduce the bug you can use `curl` to test the REPORT request with the XML data found [here](https://gist.github.com/Ede123/18def2c7a6b83d776339):
- Run `curl -X REPORT -d @multiget.xml http://localhost:5232/user/calendar.ics/`
  - In the above request you have to set the hostname / username / calendar name according to your Radicale set-up
  - In the multiget.xml you have to adjust the href according to your Radicale set-up (e.g. replace hostname / username / calendar name like above and replace the event's UID with an UID that is actually available on your server)

Then you can test multiple variations:
- With the initial href (a valid absolute URL, in my case "http://localhost:5232/user/calendar.ics/1397689200000-975832323.ics") Radicale answers with an empty MULTISTATUS
- With a multitude of different (and even wrong) hrefs Radicale will however return the ETAG and calendar data of the event correctly. For example all of the following work:
  - /1397689200000-975832323.ics
  - /calendar.ics/1397689200000-975832323.ics
  - /user/calendar.ics/1397689200000-975832323.ics
  - /http://localhost:5232/user/calendar.ics/1397689200000-975832323.ics
  - /localhost:5232/user/calendar.ics/1397689200000-975832323.ics
  - ////http://///localhost:5232//user////calendar.ics////1397689200000-975832323.ics//// (yeah... you get it...)
  - ////this/is/a/joke////1397689200000-975832323.ics//// (I liked this one)
- basically everything starting with a slash seems to work, no matter how crappy the actual href is and as long as the correct UID is the last element
- however if the initial slash is missing it doesn't work, no matter what href you specify

**System info:**
- I confirmed this issue in Radicale 0.8, as well as 0.9b1 and the latest 0.9b2
- I confirmed this issue on Windows XP Prof. as well as Windows 7 Prof. x64
- I confirmed this issue with Python 3.3.3 and Python 3.4.0/3.4.1
